assets/js/views/App.vue

```diff
@@ -114,6 +151,40 @@ export default {
 				currency: this.store.state.currency,
 			};
 		},
+		logoutCount() {
+			let count = 0;
+
+			this.providerLogins.forEach(function (login) {
+				if (!login.loggedIn) {
+					count++;
+				}
+			});
+
+			return count;
+		},
+		providerLogins() {
+			let providerlogins = [];
+			this.store.state.loadpoints.forEach(function (lp) {
+				if (lp.vehicleProviderLoginPath && lp.vehicleProviderLogoutPath) {
+					console.log(
+						"Vehicle: " +
+							lp.vehicleTitle +
+							": " +
+							lp.vehicleProviderLoginPath +
+							" -> " +
+							lp.vehicleProviderLogoutPath
+					);
+					providerlogins.push({
+						title: lp.vehicleTitle,
+						loggedIn: lp.vehicleProviderLoggedIn,
+						loginPath: lp.vehicleProviderLoginPath,
+						logoutPath: lp.vehicleProviderLogoutPath,
+					});
+				}
+			});
+
+			return providerlogins;
```

Comment:
Hier die funktionalere Kurzversion. Ich hoffe ich hab alle Klammern korrekt :D

```suggestion
			return this.store.state.loadpoints
			    .filter((lp) => lp.vehicleProviderLoginPath && lp.vehicleProviderLogoutPath)
			    .map((lp) => ({
						title: lp.vehicleTitle,
						loggedIn: lp.vehicleProviderLoggedIn,
						loginPath: lp.vehicleProviderLoginPath,
						logoutPath: lp.vehicleProviderLogoutPath,
			    }))
```